lab_2_keywords_cooccurrence/main.py

```diff
@@ -141,7 +243,17 @@ def calculate_cumulative_score_for_candidates_with_stop_words(candidate_keyword_
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not check_list(candidate_keyword_phrases, tuple, False) or not check_list(stop_words, str, False) \
+            or not check_dict(word_scores, str, Union[int, float], False):
+        return None
+    candidates_cumulative_score = {}
+    for phrase in candidate_keyword_phrases:
+        candidates_cumulative_score[phrase] = 0
+        for word in phrase:
+            if word not in stop_words:
+                candidates_cumulative_score[phrase] += int(word_scores[word])
```

Comment:
`int(word_scores[word])` can you explaint turning it to int?

Reply:
fixed